eggdrop.conf

```diff
@@ -1158,17 +1158,21 @@ server add ssl.example.net +7000
 #set sasl-timeout 15
 
 # To request the account-notify feature via CAP, set this to 1
-#set account-notify 0
+set account-notify 1
 
 # To request the extended-join feature via CAP, set this to 1
-#set extended-join 0
+set extended-join 1
 
 # To request the invite-notify feature via CAP, set this to 1
 #set invite-notify 0
 
-# To request the message-tags feature via CAP, set this to 1. NOTE: Enabling
-# this feature may interfere with RAW binds in scripts.
-#set message-tags 0
+# To request the message-tags feature via CAP, set this to 1
```

Comment:
Does this no really longer affect RAW binds after these changes? IIRC, a rawt bind will stop, and the raw bind table will not be checked?

Also, do we really want this enabled by default in a general sense? I suspect you did it for account-tag, but ... enh. Change.

Reply:
can you find out if we need it or if it works separately from account-tag? I disabled it for now and only set account-tag

Reply:
account-tag will insert a tag to a message, even if message-tags is not enabled.